### PR TITLE
ENG-11037 move partition leaders and export masters before a node is stopped.

### PR DESCRIPTION
**Pull request by zeemanhe**
voltadmin stop command invokes  @PrepareStopNode to kick off the process of moving partition leaders and export masters away from the node to be shutdown. After all is done,  invoke @StopNode to shutdown the node.  Using --force option for immediate shutdown without triggering leader and export master migration.

### CODE REVIEW

File: lib/python/voltcli/voltadmin.d/stop.py [hunk 1]
```diff
@@ -63,6 +69,34 @@ def stop(runner):
                           runner.opts.username, runner.opts.password,
                           runner.opts.ssl_config, runner.opts.kerberos)
 
+    if not runner.opts.forcing:
+        stateMessage = 'The node shutdown process has stopped.'
+        actionMessage = 'You may shutdown the node with the "voltadmin stop --force" command.'
+        try:
+            runner.info('Preparing for stopping node.')
+            resp = runner.call_proc('@PrepairStopNode',
```
Comment (rdykiel) [inline lib/python/voltcli/voltadmin.d/stop.py]: '@PrepairStopNode' should be '@PrepareStopNode'

Comment (rdykiel) [reply lib/python/voltcli/voltadmin.d/stop.py]: Must fix this everywhere.

File: src/frontend/org/voltdb/InvocationDispatcher.java [hunk 2]
```diff
@@ -420,6 +422,10 @@ else if ("@StopNode".equals(procName)) {
                 CoreUtils.logProcedureInvocation(hostLog, user.m_name, clientInfo, procName);
                 return dispatchStopNode(task);
             }
+            else if ("@PrepairStopNode".equals(procName)) {
```
Comment (rdykiel) [inline src/frontend/org/voltdb/InvocationDispatcher.java]: Fix also

File: src/frontend/org/voltdb/InvocationDispatcher.java [hunk 3]
```diff
@@ -754,6 +760,51 @@ private ClientResponseImpl dispatchStopNode(StoredProcedureInvocation task) {
         return new ClientResponseImpl(ClientResponse.SUCCESS, new VoltTable[0], "SUCCESS", task.clientHandle);
     }
 
+    private ClientResponseImpl dispatchPrepairStopNode(StoredProcedureInvocation task) {
```
Comment (rdykiel) [inline src/frontend/org/voltdb/InvocationDispatcher.java]: Fix method name

File: src/frontend/org/voltdb/iv2/InitiatorMailbox.java [hunk 4]
```diff
@@ -388,11 +388,6 @@ private void initiateSPIMigrationIfRequested(Iv2InitiateTaskMessage msg) {
             return;
         }
```
Comment (rdykiel) [inline src/frontend/org/voltdb/iv2/InitiatorMailbox.java]: Why is this removed?

File: tests/frontend/org/voltdb/regressionsuites/TestStopNode.java [hunk 5]
```diff
@@ -357,6 +359,41 @@ public void testMixStopNodeWithNodeFailure() throws Exception {
         assertFalse(lostConnect);
     }
 
+    public void testPrepairStopNode() throws Exception {
```
Comment (rdykiel) [inline tests/frontend/org/voltdb/regressionsuites/TestStopNode.java]: Fix typos here also

File: lib/python/voltcli/voltadmin.d/stop.py (hunk 1 above)
Comment (zeemanhe) [reply lib/python/voltcli/voltadmin.d/stop.py]: fixed
